Yoast/yoast-acf-analysis
# QAK-2538 Fix error on term pages

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where ACF Analysis would fail on term pages.

## Relevant technical choices:

* This is a load order problem. In the `term-scraper` of Yoast SEO we still use `jQuery( document ).ready`. 100ms should be enough to run after the term-scraper initialization. Alternatively we could remove jQuery ready there. Resulting in both "only" using WP's `domReady`, which would work too.

## Test instructions

This PR can be tested by following these steps:

1. Install and activate the following plugins on WordPress 5.6: Yoast SEO 15.4, ACF Content Analysis for Yoast SEO 3.0 and Advanced Custom Fields 5.9.3
2. Configure ACF to show a custom text field on taxonomy forms (see screenshot in issue)
3. Open the browser console and visit a taxonomy edit page
4. Notice the error is no longer in the console.

Fixes https://yoast.atlassian.net/browse/QAK-2538
